pkg/commands/isrunning/isrunning_test.go

```diff
@@ -0,0 +1,136 @@
+package isrunning_test
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudtrail"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/configuration-anomaly-detection/pkg/commands/isrunning"
+	"github.com/openshift/configuration-anomaly-detection/pkg/commands/isrunning/mock"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+)
+
+var _ = Describe("Isrunning", func() {
+	Describe("AreAllInstancesRunning", func() {
+		/*
+			- get CHGM alert
+			- extract clusterid
+			- check by infra id if all instances are running
+			- if false, check if the action was done by an authorized user
+		*/
+		var fakeErr = fmt.Errorf("test")
+		var (
+			mockCtrl          *gomock.Controller
+			mockClient        *mock.MockIsRunningService
+			isRunning         isrunning.IsRunning
+			cluster           *v1.Cluster
+			clusterDeployment hivev1.ClusterDeployment
+			infraID           string
+			instance          ec2.Instance
+		)
+		BeforeEach(func() {
+			mockCtrl = gomock.NewController(GinkgoT())
+			mockClient = mock.NewMockIsRunningService(mockCtrl)
+			isRunning = isrunning.IsRunning{IsRunningService: mockClient}
+			var err error
+			cluster, err = v1.NewCluster().ID("12345").Build()
```

Comment:
when I tried to merge these two lines I got an issue, this is the solution so far and I am glad it works :)

Reply:
basically a self review note, in case someone wants to comment against it